features/Steps/AuthorizationSteps.php

```diff
@@ -24,6 +26,15 @@ public function iAmUsingTheRDFBaseURL(): void
         $this->requestState->setBaseUrl($this->config['base_url_rdf']);
     }
 
+    /**
+     * @Given I am using the UiTiD base URL
+     */
+    public function iAmUsingTheUiTiDBaseURL(): void
+    {
+        $this->variableState->setVariable('baseUrl', $this->config['base_url_uitid']);
```

Comment:
`base_url_uitid` is missing from https://github.com/cultuurnet/appconfig/blob/main/files/udb3/docker/udb3-backend/features/config.features.php So I get errors, while I run these tests

Reply:
With the updated `config.features.php`, it works.

Reply:
I agree it should be moved to the appconfig repository once everything works.